runtime/lib/backend/CMakeLists.txt

```
@@ -1,10 +1,11 @@
-if(ENABLE_CLANG_TIDY)
+if(RUNTIME_CLANG_TIDY)
 if(NOT DEFINED CLANG_TIDY_BINARY)
-    set(CLANG_TIDY_BINARY clang-tidy)
+    set(CLANG_TIDY_BINARY clang-tidy-13)
```

If we want to be consistent with how we deal with our `clang-format` dependency, we should add `clang-tidy==14.*` to our `requirements.txt` and use that by default:
```suggestion
    set(CLANG_TIDY_BINARY clang-tidy)
```

---

The requirements.txt is missing